server/files/entrypoint_nginx.sh

```diff
@@ -104,6 +104,7 @@ sync_files(){
     for DIR in $(ls /var/www/MISP/app/files.dist); do
         rsync -azh --delete "/var/www/MISP/app/files.dist/$DIR" "/var/www/MISP/app/files/"
     done
+    rsync -azh --delete "/var/www/MISP/app/webroot/img/orgs.dist/" "/var/www/MISP/app/webroot/img/orgs"
```

Comment:
I think the `--delete` here will delete any org images uploaded by the user - can you verify that?

Reply:
I verified and of course the --delte flag was wrong. Fixed it.